src/sonic-yang-models/yang-models/sonic-bgp-neighbor.yang

```diff
@@ -97,6 +97,11 @@ module sonic-bgp-neighbor {
                     }
                     description "Route reflector client";
                 }
+
+                leaf admin_status {
+                    type boolean;
```

Comment:
I believe you should use the sonic type admin status as the values for the field will be 'up'/'down'
https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-types.yang#L54

Reply:
leaf admin_status {
					type stypes:admin_status;
				}

Reply:
> I believe you should use the sonic type admin status as the values for the field will be 'up'/'down' https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-types.yang#L54

Good suggestion!
I used the same type as sonic-bgp-common.yang, and please refer to BGP_NEIGHBOR_LIST.
https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-bgp-common.yang#L316-L319
Would you please confirm which type should be used?

Reply:
https://github.com/Azure/sonic-buildimage/blob/master/dockers/docker-fpm-frr/frr/bgpd/bgpd.spine_chassis_frontend_router.conf.j2

Per the above bgpcfgd module (refer above code), we should use 'up'/'down', please make the changes.

Reply:
@venkatmahalingam 
https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-yang-models/yang-models/sonic-bgp-common.yang#L316-L319
How about sonic-bgp-common.yang? I suppose it should use boolean type?